### PR TITLE
google => Add support for rendering (but not configuring) AutoHealer …

### PR DESCRIPTION
**Pull request by duftler**
…(alpha) maxUnavailable setting.

### CODE REVIEW

**Comment by duftler** [comment]
@danielpeach please review.

File: app/scripts/modules/google/serverGroup/details/serverGroupDetails.gce.controller.js [hunk 1]
```diff
@@ -170,6 +170,14 @@ module.exports = angular.module('spinnaker.serverGroup.details.gce.controller',
 
         this.serverGroup.autoHealingPolicyHealthCheck = healthCheckUrl ? _.last(healthCheckUrl.split('/')) : null;
         this.serverGroup.initialDelaySec = autoHealingPolicy.initialDelaySec;
+
+        if (autoHealingPolicy.maxUnavailable) {
+          if (autoHealingPolicy.maxUnavailable.percent) {
```
Comment (danielpeach) [inline app/scripts/modules/google/serverGroup/details/serverGroupDetails.gce.controller.js]: Can percent be 0?

Comment (duftler) [inline app/scripts/modules/google/serverGroup/details/serverGroupDetails.gce.controller.js]: Got this from the javadoc: `percent must belong to [0, 100]`

Comment (duftler) [inline app/scripts/modules/google/serverGroup/details/serverGroupDetails.gce.controller.js]: Oh, I think I see your point now.

Comment (duftler) [inline app/scripts/modules/google/serverGroup/details/serverGroupDetails.gce.controller.js]: Fixed, thanks.

**Comment by danielpeach** [comment]
LGTM

**Pull request merged**